app/javascript/App/bundles/ExchangeListings/components/Tips.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import Icon from '../../../components/Icon'
+import chartIcon from '../../../images/chartIcon.svg'
+import filterIcon from '../../../images/filterIcon.svg'
+import listIcon from '../../../images/listIcon.svg'
+
+const titleStyle = {
+  fontSize: '1.13rem',
+  fontWeight: 'bold',
+}
+
+titleStyle.paddingLeft = 15
+titleStyle.paddingBottom = 10
+
+const listTextNodeStyle = { width: '90%', marginLeft: 10, fontSize: '.88rem' }
+
+const listNumberStyle = {
+  background: 'rgb(47, 174, 237)',
+  borderRadius: 40,
+  color: '#fff',
+  width: 18,
+  paddingLeft: 6,
+  paddingTop: 2,
+  fontSize: 10,
+  verticalAlign: 'top',
+  marginTop: 2,
+}
+
+export default ({ closeTips, user, loggedIn }) => {
+  const userIsLoggedIn = !!(user || loggedIn)
```

Comment:
How it can be possible that `loggedIn` is `true` but no `user` in the state (I'm not familiar w/ domain but in general it's not possible).